### ISSUE TITLE
OpenGL Bug

### ISSUE THREAD

**Issue by OpenGlBUG** [hrydgard/ppsspp#8041]
It says that my GPU support OpenGL 3.1...
http://i.imgur.com/AKfIkVz.png
But PPSSPP says that my GPU dont support nor 2.0
http://i.imgur.com/jwLdVk6.png

And i don't know how to report it technically.

**Comment by Bigpet** [comment]
update your GPU driver

**Comment by sum2012** [comment]
What is your windows verson ? (XP,Vista ,8,etc)

**Comment by OpenGlBUG** [comment]
Windows 10 Pro, sir!

**Comment by sum2012** [comment]
Just confirm one thing
Does v1.0.1-809-gee2238d work
and  v1.0.1-811-g14fd313 don't work
(Download from  http://buildbot.orphis.net/ppsspp/index.php?m=fulllist )

**Comment by OpenGlBUG** [comment]
Yep! v1.0.1-809-gee2238d and any previous version open normally! And v1.0.1-811-g14fd313 and all later versions says that my GPU don't support GL 2.0 :/
And yes versions downloaded from this buildbot.

**Comment by sum2012** [comment]
PPSSPP support windows 10 not very good :\

**Comment by Bigpet** [comment]
I run Windows 10 on a PC with an Nvidia GPU and a Laptop with just an Intel i5 3230M (HD4000) and I cannot reproduce this with either of them

**Comment by unknownbrackets** [comment]
If #7897 did this, then it means your driver is acting differently with Windows 10 software.  Could be the driver doesn't actually support Windows 10 properly, so enabling Windows 10 support in Windows breaks it.

Compatibility mode is probably your best bet until the driver is updated.

-[Unknown]

**Comment by pal1000** [comment]
@Bigpet, you are not affected because this issue only affects Sandy Bridge and bellow. That's because Intel never released drivers for Windows 10 and most likely it would never do so. See: https://communities.intel.com/thread/61432

@unknownbrackets, running PPSSPP in compatibility mode doesn't help. I tested this on a Dell Vostro 2521 with Intel Core i3-2375M and ntel HD Graphics 3000. 
Reverting 98d1e5f makes OpenGL work again.

**Comment by unknownbrackets** [comment]
Should we just revert it then?  Downside is that we will show "Windows 8" or something instead of "Windows 10" in system info.  May have impact on other drivers, too...

Not sure if there's any runtime way to deal with this.

-[Unknown]

**Comment by Bigpet** [comment]
@unknownbrackets yeah, I don't think we can change it at runtime, so revert has my vote until Win10 compat is useful for something.

**Comment by hrydgard** [comment]
Unfortunate, but yeah, we can revert it.
